docs/_static/spec/rasa.yml

```diff
@@ -736,6 +737,19 @@ components:
         type: number
         default: None
       required: false
+    output_channel:
+      in: query
+      name: output_channel
+      description: >-
+        The bot's utterances will be forwarded to this channel. It uses the credentials
+        of the input channel to connect. In case the channel does
```

Comment:
`It uses the credentials of the input channel to connect.` - So output and input channel need to be the same? I would suggest writing something like `It uses the credentials listed in 'credentials.yml' to connect`.